### DISCUSSION TITLE
Help with recording

### DISCUSSION THREAD

**Discussion by jrem090** [freemocap/freemocap#147] [Q&A]
I've setup my cameras, and they can see the boards no problem.

Whenever I run however the raw_videos are empty even though I see output while running and the tables after ending the recording (by hitting esc)

I do get issues when using the divx and h264 codec. It always downgrades me to mp4v.

When It gets to the synchronize stage i get the following error
![image](https://user-images.githubusercontent.com/3639810/172905213-c4598ea2-72b4-4ac9-bf1b-5f2ea1fd33ce.png)

Please advise

**Comment by jrem090** [comment]
Seems to be working better. I downgraded protobuf instead of setting the protobuf implementation to python

**Reply by alexiscatnip** [reply]
I have the same error - video files are empty - , how did you downgrade protobuf?

**Reply by alexiscatnip** [reply]
I'm on windows 10

**Reply by jrem090** [reply]
While in the freemocap-env 
pip install protobuf==3.19.*

This should uninstall the current version and give you the latest minor version of 3.19

**Reply by jonmatthis** [reply]
Thanks for figuring this out! It was super cool  to come look at this issue and see that another member of the community had already come along to help 😊

I'm guessing some upgrade to `protobuf` broke something in our software. I'll pin the version in the next update.

**Reply by ghost** [reply]
This helped me too, but for me things blew up at the "import freemocap" stage.

**Comment by frnsys** [comment]
If anyone else is having this issue of empty recordings, I had to change the line `out.write(frame)` to `out.write(frame_sized)` in `startcamrecording.py`. The `cv2.VideoWriter` needs to be given frames that match the frame size it was initialized with. The frames are being properly resized in the code but for some reason the resized frame wasn't being passed to the video writer.
